### PR TITLE
Issue 84

### PR DESCRIPTION
**Pull request by tripodsan**

### CODE REVIEW

File: tests/tests.js [hunk 1]
```diff
@@ -52,6 +53,66 @@ exports.defineAutoTests = function() {
...
+         */
+        function syncAndTest(appId, success, fail) {
+            var sync = ContentSync.sync({
+                id: appId,
+                type: 'local',
+                copyRootApp: true
+            });
+            sync.on('complete', function(localDataPath) {
+                var file = appId + '/index.html';
+                window.requestFileSystem(LocalFileSystem.PERSISTENT, 0, function(fs) {
```
Comment (macdonst) [inline tests/tests.js]: I don't like the idea of checking in a test that will pass on Android but not on iOS. Can you switch this to using:

```
var file = appId + '/index.html';
window.resolveLocalFileSystemURL("file://" + data. localDataPath + "/" + file, function(fileEntry) {
    expect(fileEntry).toBeDefined();
    success();
}, function(e){
    fail(file + ' should exist in local copy. Error code ' + e.code);
});
```

That should work for both platforms but I'm writing this code freestyle so please check. We can write another test using window.requestFileSystem when we resolve issue #83.

Comment (tripodsan) [inline tests/tests.js]: well, that is this code about. it should work the same on android and ios. once you fixed the inconsistency, the test will pass. until then, we can ignore them.

Comment (macdonst) [inline tests/tests.js]: Yeah, but you don't check in failing tests. Then the CI fails. I agree that it should work the same on both iOS and Android and we have issue #83 to track that. That'll be fixed very soon but I'd still rather not to check in a failing test. It just seems wrong to me.

Comment (tripodsan) [inline tests/tests.js]: ok. I'll remove it from here... I think it got mixed up.

**Pull request merged**